docs/UserGuide.md

```diff
@@ -2,17 +2,9 @@
 layout: page title: User Guide
 ---
 
-| Quick Links                         |
-|-------------------------------------|
-| [Content Page](#Content-Page)       |
-| [Quick start](#Quick-start)         |
-| [Features](#Features)               |
-| [Command Summary](#Command-Summary) |
+**Ultimate DivocTracker _(UDT)_** is a desktop app for managing COVID-19 contacts in school administration, optimized for use via interacting with the application through easy-to-use commands through a user-friendly interface. Ultimate Divoc Tracker can get your contact-tracing tasks done faster than traditional GUI apps if you can type fast. Even if you can't, this user guide will guide you through how to use UDT effectively and efficiently.
```

Comment:
I think maybe 1 too many "through"s here. Can say "interacting with the application through easy-to-use commands on a user-friendly/centric interface.
The "type fast part" can remove I think.